src/JavaDependenciesManager.ts

```diff
@@ -19,7 +19,7 @@ import * as path from 'path';
 import * as vscode from 'vscode';
 
 const PREFERENCE_KEY_JAVA_REFERENCED_LIBRARIES = "java.project.referencedLibraries";
-export const CAMEL_VERSION = "3.5.0";
+export const CAMEL_VERSION = "3.7.0";
```

Comment:
is camel 3.7.0 required for camel k 1.3.1? if not then we are mixing concerns here.

Reply:
it is required.
This value needs to be aligned with the version that is used by Camel K.
it is currently hardcoded to match the default version used of Camel K that we support. it will be dynamic when `kamel inspect` will be working and dynamic resolution of dependencies implemented